src/components/calcite-pick-list/calcite-pick-list.tsx

```diff
@@ -104,7 +104,10 @@ export class CalcitePickList<
 
   lastSelectedItem: ItemElement = null;
 
-  mutationObserver = createObserver("mutation", mutationObserverCallback.bind(this));
+  mutationObserver = createObserver("mutation", () => {
+    this.setUpItems();
```

Comment:
This is not needed since [`mutationObserverCallback` calls this method](https://github.com/Esri/calcite-components/blob/15410a72c52a7ceb5e8031facccb763a5f8749de/src/components/calcite-pick-list/shared-list-logic.ts#L14).